ember/app/cases/detail/work-items/edit/index/template.hbs

```diff
@@ -0,0 +1,121 @@
+<h1>
+  {{t "workItems.edit"}}
+</h1>
+
+<hr>
+{{#if @model.isLoading}}
+  <div class="uk-text-center uk-padding">
+    <UkSpinner @ratio={{2}} />
+  </div>
+{{else}}
+  <h2>
+    {{this.workItem.task.name}}
+  </h2>
+
+  <div class="work-item-attributes">
```

Comment:
I don't see any definitions of those classes used in this file. This is just plain copy paste from one of our projects. Please clean it up.

Reply:
Removed